app/uk/gov/hmrc/gform/views/form/addToListCheckYourAnswers.scala.html

```diff
@@ -54,27 +60,48 @@
   new BackLink(href = uk.gov.hmrc.gform.gform.routes.FormController.backAction(formTemplate._id, maybeAccessCode, sectionNumber, fastForward).path,
                content = new Text(messages("linkText.back")))
 }
+
+@twoThirdsMainContent = @{new TwoThirdsMainContent}
+@fullMainContent = @{new FullMainContent}
+@threeQuartersMainContent = @{new ThreeQuartersMainContent}
+
+@mainContent = @{
+  contentBlock: Html => displayWidth match {
+    case LayoutDisplayWidth.L  => threeQuartersMainContent(contentBlock)
+    case LayoutDisplayWidth.XL => fullMainContent(contentBlock)
+    case LayoutDisplayWidth.M  => twoThirdsMainContent(contentBlock)
+    case _                      => twoThirdsMainContent(contentBlock)
```

Comment:
Arrow are not aligned. Line 73 has one extra space

Reply:
updated PR